ampcrowd/basecrowd/models.py

```diff
@@ -150,3 +171,43 @@ def add_model_rels(self):
         # responses pertain to a task
         self.add_rel(self.response_model, self.task_model, models.ForeignKey,
                      'task', 'responses')
+
+
+class TemplateResource(models.Model):
+    name = models.CharField(max_length=200)
+    content = models.TextField()
+    direct_dependencies = models.ManyToManyField('self', symmetrical=False, related_name="upstream_dependencies")
+    direct_requirements = models.ManyToManyField('self', symmetrical=False, related_name="upstream_requirements")
```

Comment:
The names I chose for these is very poor. The idea is that a template might have
1. Dependencies it will import
2. A set of requirements in order to run

For example, suppose I have two js templates. In order for one to work, it requires the other. This would be one way to enforce this and have a way to show which templates are related to each other. The other case would be a way to include stuff like jquery/react within the system instead within the template.

Upon further thought, I think it would help if I renamed those to something like `imports` and `requirements`. Thoughts on name?